packages/patterns-react/src/patterns/LeadSpace/LeadSpace.js

```diff
@@ -5,13 +5,92 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import PropTypes from 'prop-types';
 import React from 'react';
+import classnames from 'classnames';
+import { ArrowRight20 } from '@carbon/icons-react';
+import { settings } from 'carbon-components';
+import { Button } from 'carbon-components-react';
+
+const { prefix } = settings;
+
+/**
+ * renders the buttons (max 2 buttons)
+ *
+ * @param {Array} buttons array of buttons
+ * @returns {*} button components
+ */
+function renderButtons(buttons) {
+  return buttons.map((button, key) => {
+    if (key > 1) return;
+    return (
+      <Button
+        data-autoid={`leadspace__cta-${key}`}
+        renderIcon={button.renderArrow && ArrowRight20}
+        href={button.link}
+        kind={key === 0 ? 'primary' : 'tertiary'}>
+        {button.copy}
+      </Button>
+    );
+  });
+}
 
 /**
- * Lead Space pattern
+ * Lead space component
  *
- * @returns {*} Lead Space pattern
+ * @param {object} props props object
+ * @param {string} props.variation variation of the lead space (expressive (default) or productive)
+ * @param {string} props.title lead space title
+ * @param {string} props.copy lead space short copy to support the title
+ * @param {object} props.image image object with diff source for diff breakpoints
+ * @param {Array} props.buttons array of buttons for lead space (max 2 buttons)
+ * @returns {*} Lead space component
  */
-const LeadSpace = () => <div data-autoid="leadspace">Lorem Ipsum</div>;
+const LeadSpace = ({ variation, title, copy, buttons, image }) => (
+  <section
+    data-autoid="leadspace"
+    className={classnames(`${prefix}--leadspace`, {
+      [`${prefix}--leadspace--productive`]: variation === 'productive',
+    })}>
+    <div className={`${prefix}--leadspace__container`}>
+      {image && (
+        <picture>
```

Comment:
this is a good point - we could consider building out a component that takes in any number of image sources. For the leadspace at different breakpoints we have the different aspect ratios so user will have to put in the 3 sources otherwise it ends up looking wonky.

If we were to build out a component - would that be something that would go under the `react` package?